examples/README.md

```diff
@@ -14,6 +16,70 @@ The following examples are included:
 * `PersistentPageMemoryStorageExample` - demonstrates the usage of the PageMemory storage engine configured with a persistent data region.
 * `RocksDbStorageExample` - demonstrates the usage of the RocksDB storage engine.
 
-Before running the examples, read about [cli](https://ignite.apache.org/docs/3.0.0-beta/ignite-cli-tool).
+## Running the examples with an Ignite node within a Docker container
+
+1. Open the Ignite project in your IDE of choice.
+
+2. Prepare an environment variable:
+```shell
+IGNITE_SOURCES=/path/to/ignite3-sources
+```
+
+3. Start an Ignite node:
+```shell
+docker run --name ignite3-node -d --rm -p 10300:10300 -p 10800:10800 \
+  -v $IGNITE_SOURCES/examples/config/ignite-config.conf:/opt/ignite/etc/ignite-config.conf apacheignite/ignite3
+```
+
+4. Find out IP address of the node:
+```shell
+NODE_IP_ADDRESS=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ignite3-node)
+```
+
+5. Initialize the node:
+```shell
+docker run -it apacheignite/ignite3 cli cluster init --url http://$NODE_IP_ADDRESS:10300 --name myCluster1 \
+  --cluster-management-group defaultNode --metastorage-group defaultNode
+```
+
+6. Run an example via IDE.
+
+7. Stop the Ignite node:
+```shell
+docker stop ignite3-node
+```
+
+## Running the examples with an Ignite node started natively
+
+1. Open the Ignite project in your IDE of choice.
+
+2. Download the Ignite ZIP packaging with DB and CLI parts. Or build them from Ignite sources (see [DEVNOTES.md](../DEVNOTES.md)). Unpack.
```

Comment:
2. Download the Ignite ZIP package including the database and CLI parts. Alternatively, build these parts from the Ignite sources (see [DEVNOTES.md](../DEVNOTES.md)). Unpack.